src/shared/messages.ts

```diff
@@ -22,6 +23,7 @@ export type MessageHKT<
   ? { action: TAction }
   : { action: TAction; payload: TPayload }
 
+// #region Popup ↦ BG
```

Comment:
Aside: the region comments look like this in VS Code:
![image](https://github.com/interledger/web-monetization-extension/assets/8426945/ec2bba18-70c2-40a0-850e-8915db11198d)
(and regions can be collapsed/folded)